### PR TITLE
Div campaign 2022

### PR DESCRIPTION
**Pull request by abelsromero**
WIP: pending decision for applications channel

### CODE REVIEW

**Comment by anyulled** [comment]
Hay un icono o emoji que no se ve aquí, en Windows.
screenshot


<details><summary>Browser metadata</summary>





```
Path:      /2022/diversity.html
Browser:   Chrome 98.0.4758.102 on Windows 10
Viewport:  1535 x 1576 @1.25x
Language:  es-ES
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-1076--jbcnconf.netlify.app%2F2022%2Fdiversity.html&os=Windows&os_version=10&browser=Chrome&browser_version=98.0&resolution=1535x1576&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-1076--jbcnconf.netlify.app/2022/diversity.html) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6227e123acaff40008fefebd&commentID=62290c1ce42c575bbe718d71&resolution=resolved)

**Comment by anyulled** [comment]
Hay un icono o emoji que no se ve aquí. Usando Windows y Chrome
screenshot


<details><summary>Browser metadata</summary>





```
Path:      /2022/diversity.html
Browser:   Chrome 98.0.4758.102 on Windows 10
Viewport:  1535 x 1576 @1.25x
Language:  es-ES
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-1076--jbcnconf.netlify.app%2F2022%2Fdiversity.html&os=Windows&os_version=10&browser=Chrome&browser_version=98.0&resolution=1535x1576&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-1076--jbcnconf.netlify.app/2022/diversity.html) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6227e123acaff40008fefebd&commentID=62290d0f53c96e5d13abdd94&resolution=resolved)

File: 2022/_includes/diversity/diversity-terms.html [hunk 1]
```diff
@@ -7,33 +7,45 @@
 People who are diverse in nationality, age, race, ethnic origin, spiritual beliefs and practices, gender, gender identity, sexual orientation, work and life status are critical for constant improvement and innovation in our profession.</p>
 </div>
 <div class="paragraph">
-<p>That why at JBCNConf we are offering diversity a plan.</p>
+<p>That&#8217;s why we are running again the <em>diversity a plan for JBCNConf 2022</em> 🎉</p>
 </div>
 <div class="paragraph">
 <p>If you belong to a traditionally underrepresented group in the tech community, now is your chance.
-We definitely want you to come and join us at the conference.</p>
+We definitely want you to come and join us at the conference 🫂</p>
```
Comment (anyulled) [inline 2022/_includes/diversity/diversity-terms.html:14]:
```suggestion
We definitely want you to come and join us at the conference</p>
```

Comment (abelsromero) [reply 2022/_includes/diversity/diversity-terms.html:14]: Lo de la sugestión es automático?

Comment (abelsromero) [reply 2022/_includes/diversity/diversity-terms.html:14]: Interestingly, it does work in MacOS. The thing is that was the only "hug-related" neutral emoji I saw, other than removing, someone has any suggestion?

Comment (anyulled) [reply 2022/_includes/diversity/diversity-terms.html:14]: I just updated my commit suggestion  with the hug emoji from https://emojipedia.org/hugging-face/ you can look for any other as well

Comment (abelsromero) [reply 2022/_includes/diversity/diversity-terms.html:14]: Thats the thing, already did and that one points to and does not work with my Linux fonts
image...I love multi-platforms 🌈

Comment (abelsromero) [reply 2022/_includes/diversity/diversity-terms.html:14]: A falta de otro, aplico 🤗

**Review by anyulled — APPROVED** [review]

**Comment by jonathanvila** [comment]
Deberia aparecer el logo de GFT que es quien paga los diversity tickets, y con un thank you y link a su pagina

**Comment by abelsromero** [comment]
He añadido una linia y el logo en debajo de la intro para q se vea bien

image

**Review by jonathanvila — APPROVED** [review]
LGTM 👍

**Pull request merged**